akiraux/Akira
# Update saving and loading of files

## Summary / How this PR fixes the problem?
This PR re-enables the saving and loading of files with the new Components architecture.

## Steps to Test
- Create a bunch of items, artboards, and import images.
- Save the file and close.
- Open Akira again and load the saved file.
- Confirm that everything was kept.

..or, you can download this file, unzip it, and open it: [test-file.zip](https://github.com/akiraux/Akira/files/6017240/test-file.zip)

## Screenshots 
This is how the file should look after being opened:
![Screenshot from 2021-02-20 21 20 11](https://user-images.githubusercontent.com/2527103/108616665-48607f80-73c4-11eb-83e0-40b37415d30b.png)

## Known Issues / Things To Do
- Multiple colors don't blend properly, yet. I'll deal with it in a dedicated PR.

## This PR fixes/implements the following **bugs/features**:
- Decreases the amount of properties in the saved file, relying only on components.
- Properly restores all the items in the correct position.
- Initial handle of multiple fill and border colors (items are not colored correctly yet).
- Hover effect on the Artboard works when hovering on the artboard layer.
- Bind the artboard's label visibility to its parent.
